.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [ $default-branch ]
+  release:
+    types:
+      - created
+
+jobs:
+  test:
+    name: CI
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp: [21.3.8.18, 22.0.7, 22.1.8, 22.2.8, 22.3.4.14, 23.0.4, 23.1.5.0, 23.2.1.0]
```

Comment:
Are all the minor versions needed?

Reply:
yes and no, I have locally some problems with a specific version. The lists makes sure that all recent minor version work.
But we might indeed reduce this a bit.